src/genie/libs/parser/iosxe/tests/test_show_cts.py

```diff
@@ -48,6 +48,165 @@ def test_show_cts_pacs_full(self):
+                    "dead_time_secs": 20
+                },
+                2: {
+                    "server_ip": "10.1.100.5",
+                    "port": 1812,
+                    "aid": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                    "server_status": "ALIVE",
+                    "auto_test": "FALSE",
+                    "keywrap_enable": "FALSE",
+                    "idle_time_mins": 60,
+                    "dead_time_secs": 20
+                },
+                3: {
+                    "server_ip": "10.1.100.6",
+                    "port": 1812,
+                    "aid": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                    "server_status": "ALIVE",
+                    "auto_test": "FALSE",
+                    "keywrap_enable": "FALSE",
+                    "idle_time_mins": 60,
+                    "dead_time_secs": 20
+                },
+                4: {
+                    "server_ip": "10.1.100.6",
+                    "port": 1812,
+                    "aid": "A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A1A",
+                    "server_status": "ALIVE",
+                    "auto_test": "FALSE",
+                    "keywrap_enable": "FALSE",
+                    "idle_time_mins": 60,
+                    "dead_time_secs": 20
+                }
+            },
+            "security_groups": {
+                1: {
+                    "sec_group": "0-15",
+                    "sec_group_name": "Unit0"
+                },
+                2: {
+                    "sec_group": "2-12",
+                    "sec_group_name": "Unit1"
+                },
+                3: {
+                    "sec_group": "3-10",
+                    "sec_group_name": "Unit2"
+                },
+                4: {
+                    "sec_group": "4-11",
+                    "sec_group_name": "Device11"
+                },
+                5: {
+                    "sec_group": "3215-08",
+                    "sec_group_name": "K2"
+                },
+                6: {
+                    "sec_group": "9999-06",
+                    "sec_group_name": "Q1"
+                },
+                7: {
+                    "sec_group": "68-10",
+                    "sec_group_name": "North"
+                },
+                8: {
+                    "sec_group": "5016-00",
+                    "sec_group_name": "Quarantine"
+                },
+                9: {
+                    "sec_group": "8000-00",
+                    "sec_group_name": "TEST_8000"
+                }
+            },
+            "env_data_lifetime_secs": "86400",
+            "last_update": {
+                "date": "Tue, Jul/21/2020",
+                "time": "20:04:42",
+                "time_zone": "PDT"
+            },
+            "expiration": "0:00:46:51",
+            "refresh": "0:00:46:51",
+            "state_machine_status": "running"
+        }
+    }
+
+    golden_output1 = {'execute.return_value': '''
+CTS Environment Data
```

Comment:
can you indent the info in this dict

Reply:
@ahersi2-cisco, to indent this, I'll have to do an lstrip against this data and test. This may cause some inaccurate testing if we're not testing against the original data.

Reply:
This has been updated.